### ISSUE TITLE
Scene Actions on Apple Watch are disabled after every Home Assistant restart.

### ISSUE THREAD

**Issue by rhyswaywood** [home-assistant/iOS#2811]
**iOS device model, version and app version**
Model Name: iPhone 13 Pro / Apple Watch Series 7
Software Version: iOS Version 17.5.1 / watchOS 10.5
App version: 2024.6 (2024.693) (TestFlight Beta)

**Home Assistant Core Version**
2024.6.1

**Describe the bug**
Every time Home Assistant needs to be restarted (ie. for updates) I lose all the Scene Actions that are enabled on my Apple Watch. I then need to open the Companion App settings / Actions to re-enable the Scene Actions I wish to display. 

**To Reproduce**
All of my Scene Actions get disabled after every restart of Home Assistant.

**Expected behavior**
The selected Scene Actions should persist across restarts.

**Additional context**
The scenes I use are defined within Zigbee2MQTT and are not native Home Assistant scenes.

**Comment by pezzy-o** [comment]
I'm seeing the same behaviour in the iOS app - scene actions have to be re-enabled after HA restart (they don't seem to maintain their enabled state from pre-restart).
Version:	core-2024.5.5
Installation Type:	Home Assistant Container

**Comment by mbarcia** [comment]
+1

**Comment by pezzy-o** [comment]
Appears to be fixed with iOS app version 2024.6.

**Comment by DaniMol** [comment]
> Appears to be fixed with iOS app version 2024.6.

No it hasn’t. Still deselecting all actions after every reboot. 

iPhone 15 pro, iOS 17.6, Home Assistant Core 2024.8.0, Companion app 2024.7

**Comment by rhyswaywood** [comment]
Yes it is also still an issue for me. All scenes get disabled after every Home Assistant reboot. 
I'm now on Core 2024.8.0
iOS App Version 2024.8 (2024.762 TestFlight beta)

I ended up creating Actions for all of my scenes instead, which just activate the scene as an automation.

**Comment by DaniMol** [comment]
> Yes it is also still an issue for me. All scenes get disabled after every Home Assistant reboot. I'm now on Core 2024.8.0 iOS App Version 2024.8 (2024.762 TestFlight beta)
> 
> I ended up creating Actions for all of my scenes instead, which just activate the scene as an automation.

My biggest problem is that I use iOS shortcuts to quickly activate scenes from my home screen. Will that work? And if so, how?

**Comment by pm-melo** [comment]
> > Yes it is also still an issue for me. All scenes get disabled after every Home Assistant reboot. I'm now on Core 2024.8.0 iOS App Version 2024.8 (2024.762 TestFlight beta)
> > I ended up creating Actions for all of my scenes instead, which just activate the scene as an automation.
> 
> My biggest problem is that I use iOS shortcuts to quickly activate scenes from my home screen. Will that work? And if so, how?

It does work, however I find that doing so loses the _snapiness_ of the process. In my case, as the HA server is exposed via a reverse proxy and I'm not using an internal DNS server (so the DNS resolution happens outside of my network), every time I trigger an automation through the use of a webhook, for example, the latency is visible.
